tutor/specs/screens/task/reading.spec.js

```diff
@@ -8,6 +8,11 @@ describe('Reading Tasks Screen', () => {
 
   beforeEach(() => {
     const task = Factory.studentTask({ type: 'reading' });
+    history = {
+      push: (url) => {
+        props.ux.goToStep(Number(ld.last(url.split('/'))) - 1, false);
```

Comment:
I think this fails if the url ends in `/`?

Reply:
yeah, it probably would but this is just a hacked together mock for the spec so it's ok.  It won't have to deal with stuff like code would "in the wild"